static/scss/videoDetail.scss

```diff
@@ -24,3 +24,73 @@
     }
   }
 }
+
+.videoSubtitles {
+  padding: 10px;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 100%;
+  align-items: stretch;
+
+  .videoSubtitle-card {
+    margin: 0 0 40px;
+    background-color: #ffffff;
+    width: 100%;
+
+    @include breakpoint(mobile) {
+      width: 100%;
+    }
+    @include breakpoint(phone) {
+      width: 100%;
+      margin: 0 0 25px;
+    }
+
+    &:hover {
+      box-shadow: 7px 8px 28px #c7c7c7;
+    }
+
+    .card-contents {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .videoSubtitle-card-body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      padding: 10px 15px 14px;
+
+      h4 {
+        flex-shrink: 1;
+        margin: 0;
+        padding: 8px;
+        line-height: 1.3rem;
+        overflow: hidden;
+        font-weight: 400;
+
+        a {
+          color: rgba(0,0,0,.87);
+        }
+      }
+
+      .actions {
+        color: #9a9a9a;
+
+        * {
+          margin-left: 10px;
+          margin-top: 0px;
+          margin-bottom: 5px;
+        }
+
+        div {
+          margin-left: 0px;
+        }
+
+        a {
+          cursor: pointer;
+        }
+      }
+    }
+  }
+}
```

Comment:
Looks like most of these style definitions were copied directly from `.videos` and `.video-card` in `collection.scss`. That's ok for now, but at the very least we should open an issue to define some common styles that both components can use

Reply:
Yes, I started with the video-card styles as a template and then modified it as best I could to resemble Robert's mockups for subtitles.

https://github.com/mitodl/odl-video-service/issues/260